src/main/kotlin/click/seichi/gigantic/message/messages/menu/SpecialThanksMenuMessages.kt

```diff
@@ -0,0 +1,14 @@
+package click.seichi.gigantic.message.messages.menu
+
+import click.seichi.gigantic.message.LocalizedText
+import org.bukkit.ChatColor
+import java.util.*
+
+object SpecialThanksMenuMessages {
+
+    val OFFICAL_SPECIAL_THANKS_DISPLAY_NAME = 
+        LocalizedText(
+            Locale.JAPANESE to "seichi.click network 整地鯖(春) Special Thanks"
```

Comment:
```suggestion
            Locale.JAPANESE to "整地鯖(春) Special Thanks"
```

のほうが短くていいかもしれない！
seichi.click networkあったほうがいいのかなぁ...

Reply:
私も短いほうが良いかなとも思ったんですけど、
コミュニティグループとしては`seichi.click network`、運用していたプロジェクトとしては`整地鯖(春)`みたいな
文言が記載してあったほうが明確に公式と非公式で区別できる部分はあったほうが良いかもしれない

Reply:
なら変更無しで大丈夫です！